web/templates/recipient_row.hbs

```diff
@@ -57,19 +57,22 @@
                 {{/if}}
             {{/if}}
 
-            {{#if topic_muted}}
-                <i class="zulip-icon zulip-icon-mute on_hover_topic_unmute recipient_bar_icon" data-stream-id="{{stream_id}}" data-topic-name="{{topic}}" data-tooltip-template-id="topic-unmute-tooltip-template" role="button" tabindex="0" aria-label="{{t 'Unmute topic' }}"></i>
-                <template id="topic-unmute-tooltip-template">
-                    {{t "Unmute topic" }}
-                    {{tooltip_hotkey_hints "Shift" "M"}}
-                </template>
+            {{#if stream_muted}}
+                <i class="zulip-icon zulip-icon-unmute stream_muted {{#if topic_unmuted_in_muted_stream}} on_hover_topic_mute {{else}} on_hover_topic_unmute {{/if}} recipient_bar_icon hidden-for-spectators" data-stream-id="{{stream_id}}" data-topic-name="{{topic}}" role="button" tabindex="0"
+                  {{#if topic_unmuted_in_muted_stream}} data-tooltip-template-id="topic-mute-tooltip-template" aria-label="{{t 'Mute topic' }}" {{else}} data-tooltip-template-id="topic-unmute-tooltip-template" aria-label="{{t 'Unmute topic' }}" {{/if}}></i>
             {{else}}
-                <i class="zulip-icon zulip-icon-mute on_hover_topic_mute recipient_bar_icon hidden-for-spectators" data-stream-id="{{stream_id}}" data-topic-name="{{topic}}" data-tooltip-template-id="topic-mute-tooltip-template" role="button" tabindex="0" aria-label="{{t 'Mute topic' }}"></i>
-                <template id="topic-mute-tooltip-template">
-                    {{t "Mute topic" }}
-                    {{tooltip_hotkey_hints "Shift" "M"}}
-                </template>
+                <i class="zulip-icon zulip-icon-mute stream_unmuted{{#if topic_muted}} on_hover_topic_unmute {{else}} on_hover_topic_mute {{/if}} recipient_bar_icon hidden-for-spectators" data-stream-id="{{stream_id}}" data-topic-name="{{topic}}" role="button" tabindex="0"
+                  {{#if topic_muted}} data-tooltip-template-id="topic-unmute-tooltip-template" aria-label="{{t 'Unmute topic' }}" {{else}} data-tooltip-template-id="topic-mute-tooltip-template" aria-label="{{t 'Mute topic' }}" {{/if}}></i>
             {{/if}}
+
+            <template id="topic-unmute-tooltip-template">
+                {{t "Unmute topic" }}
+                {{tooltip_hotkey_hints "M"}}
+            </template>
+            <template id="topic-mute-tooltip-template">
+                {{t "Mute topic" }}
+                {{tooltip_hotkey_hints "M"}}
```

Comment:
These tooltip hints are missing the `Shift`

Reply:
it is fixed in #25266 i will rebase this PR to fix this.

Reply:
No need -- I'm going to tweak and merge since I'm doing my final fixup pass.